README.md

```diff
@@ -17,13 +17,14 @@ See full documentation on [https://async-aws.com](https://async-aws.com).
 | [async-aws/cognito-identity-provider](https://github.com/async-aws/cognito-identity-provider) | [![Latest Stable Version](https://poser.pugx.org/async-aws/cognito-identity-provider/v/stable)](https://packagist.org/packages/async-aws/cognito-identity-provider) [![Total Downloads](https://poser.pugx.org/async-aws/cognito-identity-provider/downloads)](https://packagist.org/packages/async-aws/cognito-identity-provider) | [![](https://github.com/async-aws/cognito-identity-provider/workflows/BC%20Check/badge.svg?branch=master)](https://github.com/async-aws/cognito-identity-provider/actions)
 | [async-aws/dynamo-db](https://github.com/async-aws/dynamo-db)                                 | [![Latest Stable Version](https://poser.pugx.org/async-aws/dynamo-db/v/stable)](https://packagist.org/packages/async-aws/dynamo-db)                                 [![Total Downloads](https://poser.pugx.org/async-aws/dynamo-db/downloads)](https://packagist.org/packages/async-aws/dynamo-db)                                 | [![](https://github.com/async-aws/dynamo-db/workflows/BC%20Check/badge.svg?branch=master)](https://github.com/async-aws/dynamo-db/actions)
 | [async-aws/event-bridge](https://github.com/async-aws/event-bridge)                           | [![Latest Stable Version](https://poser.pugx.org/async-aws/event-bridge/v/stable)](https://packagist.org/packages/async-aws/event-bridge)                           [![Total Downloads](https://poser.pugx.org/async-aws/event-bridge/downloads)](https://packagist.org/packages/async-aws/event-bridge)                           | [![](https://github.com/async-aws/event-bridge/workflows/BC%20Check/badge.svg?branch=master)](https://github.com/async-aws/event-bridge/actions)
-| [async-aws/iam](https://github.com/async-aws/iam)                                             | [![Latest Stable Version](https://poser.pugx.org/async-aws/iam/v/stable)](https://packagist.org/packages/async-aws/iam)                                             [![Total Downloads](https://poser.pugx.org/async-aws/iam/downloads)](https://packagist.org/packages/async-aws/iam)                           | [![](https://github.com/async-aws/event-bridge/workflows/BC%20Check/badge.svg?branch=master)](https://github.com/async-aws/event-bridge/actions)
+| [async-aws/iam](https://github.com/async-aws/iam)                                             | [![Latest Stable Version](https://poser.pugx.org/async-aws/iam/v/stable)](https://packagist.org/packages/async-aws/iam)                                             [![Total Downloads](https://poser.pugx.org/async-aws/iam/downloads)](https://packagist.org/packages/async-aws/iam)                                             | [![](https://github.com/async-aws/event-bridge/workflows/BC%20Check/badge.svg?branch=master)](https://github.com/async-aws/event-bridge/actions)
```

Comment:
We should not change this line.

I dont really see what's changed to be honest.